sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/StoreResult.java

```diff
@@ -199,6 +203,7 @@ public void serialize(StoreResult storeResult,
             jsonGenerator.writeNumberField("requestCharge", storeResult.requestCharge);
             jsonGenerator.writeNumberField("itemLSN", storeResult.itemLSN);
             jsonGenerator.writeStringField("sessionToken", (storeResult.sessionToken != null ? storeResult.sessionToken.convertToString() : null));
+            jsonGenerator.writeObjectField("backendLatencyInMs", storeResult.backendLatencyInMs);
```

Comment:
Does Java have to worry about different cultures like if a machine is set to French where it uses commas instead of decimal?

Reply:
Only if we are formatting the string to something. In this case, we are getting the double in a string from backend, which will always return the decimal value. Hence `Double.parseDouble` will work perfectly here. 
For context, we do the same with requestCharge.

Reply:
@j82w  this value coming from backend and not user data , so this is fine